Makefile

```diff
@@ -2,51 +2,54 @@
 
 APP_ID := talk_bot_ai
 APP_NAME := AssistantTalkBot
-APP_VERSION := 3.0.0
-JSON_INFO := "{\"id\":\"$(APP_ID)\",\"name\":\"Assistant Talk Bot\",\"daemon_config_name\":\"manual_install\",\"version\":\"$(APP_VERSION)\",\"secret\":\"12345\",\"port\":10034,\"scopes\":[\"ALL\"]}"
+APP_VERSION := $$(xmlstarlet sel -t -v "//version" appinfo/info.xml)
```

Comment:
Very nice! Although, this assumes that `xmlstarlet` is installed on the machine which might not be the case. Maybe it would be a good idea to require it somehow before running the rest of the Makefile.

Reply:
[imho, this should be mentioned in dev docs for AppAPI instead of implementing check in every ExApp in each Makefile](imho: this should be mentioned in dev docs for AppAPI instead of implementing check/warning in every ExApp in each Makefile)

Reply:
OK, I'll make a note of that and add it when I get back to editing the docs.